Content.Shared/Construction/ConstructionGraphNode.cs

```diff
@@ -31,6 +31,9 @@ public sealed partial class ConstructionGraphNode
         [DataField("entity", customTypeSerializer: typeof(GraphNodeEntitySerializer), serverOnly:true)]
         public IGraphNodeEntity Entity { get; private set; } = new NullNodeEntity();
 
+        [DataField("childrenIgnoreEntity")]
+        public bool ChildrenIgnoreEntity = false;
```

Comment:
comment

Reply:
See my comment below. Have renamed this to "ignoreDerivedProtoypes" and added a summery to the field to help clarity its usage